Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -211,9 +211,10 @@ internal static void ReconcileOculusIntegrationDefine(bool oculusIntegrationPres
         }
 
         /// <summary>
-        /// Adds warnings to the nowarn line in the csc.rsp file located at the root of assets.  Warning 618 and 649 are added to the nowarn line because if
+        /// Adds warnings to the nowarn line in the csc.rsp file located at the root of assets.  Warning 414, 618 and 649 are added to the nowarn line because if
```

Comment:
Do csc files work in subdirectories? Mostly, I'm wondering if we could put these warnings specifically in the root of the Oculus Integration Package folder instead of for the entire user's project, which seems a little heavy-handed.